symbolic-minidump/src/processor.rs

```diff
@@ -1206,6 +1186,65 @@ impl<'a> fmt::Debug for ProcessState<'a> {
     }
 }
 
+/// Processes a minidump supplied via raw binary data.
+////// Returns a `ProcessState` that contains information about the crashed
+/// process. The parameter `frame_infos` expects a map of Breakpad symbols
+/// containing STACK CFI and STACK WIN records to allow stackwalking with
+/// omitted frame pointers.
+pub fn process_minidump<'a>(
```

Comment:
I wonder if API-wise this makes more sense as a new method on `ProcessState`?  Maybe `ProcessState::from_minidump_with_options`?